pilot/pkg/serviceregistry/serviceentry/conversion.go

```diff
@@ -257,6 +262,12 @@ func convertServices(cfg config.Config, clusterID cluster.ID) []*model.Service {
 			},
 			ServiceAccounts: serviceEntry.SubjectAltNames,
 		}
+		if ha.autoAssignedV4 != "" {
```

Comment:
Maybe I am just missing it, but are we still setting ClusterVIPs in these cases? Won't they still get the auto-allocated IP if so?